cmd/istioctl/inject.go

```diff
@@ -159,6 +161,8 @@ func init() {
 	injectCmd.PersistentFlags().BoolVar(&enableCoreDump, "coreDump",
 		true, "Enable/Disable core dumps in injected Envoy sidecar (--coreDump=true affects "+
 			"all pods in a node and should only be used the cluster admin)")
+	injectCmd.PersistentFlags().StringVar(&imagePullPolicy, "imagePullPolicy", "Always",
+		"Sets the container image pull policy to Always or PullIfNotPresent")
```

Comment:
re: string vs. bool, the description suggests either/or behavior (e.g. bool) but its really a string value which is confusing. Maybe just update the description to include the list of valid options [Always,Never,IfNotPresent]?